code/modules/vehicles/armored/vehicle_collision.dm

```diff
@@ -0,0 +1,64 @@
+/**
+ *This proc is called when a atom is crashed into by a [armored vehicle][/obj/vehicle/sealed/armored]. Damage is then dealt to both the vehicle and atom
+ *
+ * * Arguments:
+ * * veh is the vehicle that is ramming
+ * * facing is the direction the vehicle is facing for when we ram it
+ * * T is the turf where the vehicle is used with-
+ * * temp to check whether a mob is squished
+ */
+/atom/proc/vehicle_collision(obj/vehicle/sealed/armored/veh, facing, turf/T, turf/temp)
```

Comment:
mech crushes mob under foot. Tank safely runs over mobs.
Tank = cringe

Speaking of, if you rest and let a tank roll over you, can you just get up and have an invulnerable bunker on top of you?

Reply:
yea its on the list but I wanna get this out the door first